### PR TITLE
irfan-homework

### PR DESCRIPTION
**Pull request by irfankaradeniz**
javascripthomework-irfan

### CODE REVIEW

File: Week2/irfan-homework.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+// Question 1
```
Comment (marcianoviereck) [inline Week2/irfan-homework.js:1]: Can you split up the homework in different parts:

> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference.

File: Week2/irfan-homework.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+
+// Question 5
+console.log(Math.round(7.25));
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+
+let h = Math.max(a,z);
```
Comment (marcianoviereck) [inline Week2/irfan-homework.js:37]: nice solution!

File: Week2/irfan-homework.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+let thirdVariable = [];
+let fourthVariable = true;
+
+console.log("The value of my firstVariable is: " + firstVariable);
+console.log("The value of my secondVariable is: " + secondVariable);
+console.log("The value of my thirdVariable is: " + thirdVariable);
+console.log("The value of my fourthVariable is: " + fourthVariable);
+
+
+function checkType(variable1,variable2) {
```
Comment (marcianoviereck) [inline Week2/irfan-homework.js:69]: Really nice solution!

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
The homework is fine, but can you only split it up in separate files? Then it will be approved.

**Review by marcianoviereck — APPROVED** [review]
Nice, approved!

**Comment by wouterkleijn** [comment]
Thanks @irfankaradeniz for making the necessary changes.

**Pull request closed without merging**